services/ux-backend/main.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+
+	handler "github.com/red-hat-storage/ocs-operator/v4/services/ux-backend/handlers"
+	"k8s.io/klog/v2"
+)
+
+type serverConfig struct {
+	listenPort           int
+	tokenLifetimeInHours int
+}
+
+func loadAndValidateServerConfig() *serverConfig {
+	var config serverConfig
+
+	var err error
+	defaultTokenLifetimeInHours := 48
+	tokenLifetimeInHoursAsString := os.Getenv("ONBOARDING_TOKEN_LIFETIME")
+	if tokenLifetimeInHoursAsString == "" {
+		klog.Infof("No user-defined token lifetime provided, defaulting to %d ", defaultTokenLifetimeInHours)
+		config.tokenLifetimeInHours = defaultTokenLifetimeInHours
+	} else if config.tokenLifetimeInHours, err = strconv.Atoi(tokenLifetimeInHoursAsString); err != nil {
+		klog.Errorf("Malformed user-defined Token lifetime: %s. shutting down", tokenLifetimeInHoursAsString)
+		os.Exit(-1)
+	}
+
+	klog.Infof("generated tokens will be valid for %d hours", config.tokenLifetimeInHours)
+
+	defaultListeningPort := 8080
+	listenPortAsString := os.Getenv("UX_BACKEND_PORT")
+	if listenPortAsString == "" {
+		klog.Infof("No user-defined server listening port provided, defaulting to %d ", defaultListeningPort)
+		config.listenPort = defaultListeningPort
+	} else if config.listenPort, err = strconv.Atoi(listenPortAsString); err != nil {
+		klog.Errorf("Malformed user-defined listening port: %s. shutting down", listenPortAsString)
+		os.Exit(-1)
+	}
+
+	klog.Info("ux backend server listening on port ", config.listenPort)
+	return &config
+}
+
+func main() {
+
+	klog.Info("Starting ux backend server")
+
+	config := loadAndValidateServerConfig()
```

Comment:
`loadAndValidateServerConfig` should return error as well and program flow should be handled at this point in the code.